### ISSUE TITLE
numStrides default

### ISSUE THREAD

**Issue by ARSadri** [ARSadri/RobustGaussianFittingLibrary#10]
default value for NumStrides must be zero

Also would be nice to implement multi resolution analysis

large window + 0.5 smaller + -.25 even smaller

this works better than the convolution with strides

Of course one can keep the strides

Also it would be nice to have these in C

**Comment by ARSadri** [comment]
Default is set to zero everywhere

**Comment by ARSadri** [comment]
support strideX and strideY.

**Comment by ARSadri** [comment]
No I think windowing must be done before rgflib now
